### DISCUSSION TITLE
Trying to use the fetch api

### DISCUSSION THREAD

**Discussion by gabrielchristino** [zepp-health/zeppos-forum#18] [Q&A] [answered]
Hello! I'm trying to use the Fetch API example on my GTS4 Mini, with Zepp OS 1.0
But I'm getting an error on the "App log" tab in the app, and the "HELLO ZEPPOS" didn't show on screen.
The error is : "the path of applet is invalid please check if the source of applet is correctly"
I have checked all the files, and it's all right. In the app.json, I included the GTS4 Mini in "platforms", and it's not working.

Are there anything more to do?
Thanks!!

**Comment by MyoungXUE** [comment, marked as answer]
Hello, thanks for your support to the forum.

Please also provide some more information, can you provide a screenshot of the error report?

I'll provide another idea, pull the fetchAPI 1.0 Mini Program again and give the process a smooth run on the GTR3-Pro simulator first, then simply adjust the code to add the deviceSource configuration of the GTS4 Mini and then replace the GTS4 Mini simulator to run it.

**Reply by gabrielchristino** [reply]
Hi! after a bit of code i got it to work!
Thank you so much!

**Reply by MyoungXUE** [reply]
You're welcome, have fun at Zepp OS! 😄
